server/p2p_receive_transaction_request_parser.go

```diff
@@ -27,46 +28,21 @@ func parseP2pReceiveTxRequest(c *Configuration, parms url.Values, format p2pPayl
 		incomingPaymailDomain: domain,
 	}
 
-	requestTx := paymail.P2PTransaction{}
-
-	requestTx.Reference = parms.Get("reference")
-	if len(requestTx.Reference) == 0 {
-		return nil, &parseError{ErrorMissingField, "missing parameter: reference"}
-	}
```

Comment:
Why did you remove valiadtion of required fields?

Reply:
I have no idea. Accidentally.
Thanks for noticing. I'll return it

Reply:
validations are back